custom-plugin-decorators/openai-is-consequential/README.md

```diff
@@ -8,7 +8,7 @@ Authors:
 
 OpenAI Actions uses OpenAPI to enable GPTs to make API calls.
 
-It has support for a [consequential flag](https://platform.openai.com/docs/actions/consequential-flag):
+It has support for a consequential flag (documentation no longer available, but originally at `https://platform.openai.com/docs/actions/consequential-flag`):
```

Comment:
Not sure about this change

Reply:
The link is currently broken, I have no way to know if we should keep it or not - so I did this to it for now. Alternative suggestions are welcome though!